config/routes.rb

```diff
@@ -3,4 +3,6 @@
   post "/login", to: "auth#login"
   get "/auto_login", to: "auth#auto_login"
   get "/user_is_authed", to: "auth#user_is_authed"
+
+  resources :gits
```

Comment:
should be gifts, right?

Reply:
Yes, you are correct. It is a typo. It should be fixed now